graphstorm-processing/graphstorm_processing/data_transformations/dist_transformations/dist_numerical_transformation.py

```diff
@@ -236,8 +236,8 @@ class DistNumericalTransformation(DistributedTransformation):
     def __init__(
         self,
         cols: Sequence[str],
-        normalizer: str,
-        imputer: str,
+        normalizer: str = "none",
```

Comment:
Do we address this default value in any doc?

Reply:
Yup, can see the descriptions here: https://w.amazon.com/bin/view/Amazon-Science/publishing/Overview/#HSciPubPolicyDeadlines

Reply:
Yup can see the descriptions here: https://graphstorm.readthedocs.io/en/latest/graph-construction/gs-processing/input-configuration.html#supported-transformations